common/src/main/resources/assets/gtceu/compass/pages/en_us/circuits/glass_tube.xml

```diff
@@ -1,28 +1,21 @@
 <page>
- <h1>
-  Glass Tube
- </h1>
- <h3>
-  Source
- </h3>
- <text><style color="0xaaaaaa" italic="true">Note: this page is ported from MCMOD's GTCEu wiki.<br/>Author: <style color="0xa4e7ff" hover-info="url" underlined="true" url-link="https://center.mcmod.cn/119280/">Oganesson897</style><br/></style></text>
- <h2 font-color="0xff0000">
-  WARNING!!!
- </h2>
- <text isCenter="true">
-  <style color="0xff0000">The warning means this is not a reliable page.</style>
-  <br/>
-  <style bold="true" color="0xffaa00">Please do not report any mistakes on the page, either discord or github.</style>
-  <br/>
-  This page is translated from the Chinese wiki which hasn't been reviewed and polished yet. It may contain errors and inaccuracies. We are looking forward to your help by improving this page.
- </text>
- <text>
-  <br/>
-  <style>
-   can only be used to synthesize
-   <style color="0xafff00" hover-info="click link" link="gtceu:links" underlined="true">vacuum tube</style>
-   .
-  </style>
-  <br/>
- </text>
+    <h1>
+     Glass Tube
+    </h1>
+    <h3>
+     Source
+    </h3>
+    <text><style color="0xaaaaaa" italic="true">Note: this page is ported from MCMOD's GTCEu wiki.<br/>Author: <style color="0xa4e7ff" hover-info="url" underlined="true" url-link="https://center.mcmod.cn/179043/">花海</style>, <style color="0xa4e7ff" hover-info="url" underlined="true" url-link="https://github.com/Yefancy">KilaBash</style><br/></style></text>
+    <text>
+        Glass tubes, made from glass dust in <style color="0xafff00" underlined="true" link="gtceu:machines/forming_press" hover-info="click link">Forming Press</style> or <style color="0xafff00" underlined="true" link="gtceu:machines/alloy_smelter" hover-info="click link">Alloy smelter</style>, 
+        or from glass liquid in a <style color="0xafff00" underlined="true" link="gtceu:machines/fluid_solidifier" hover-info="click link">Fluid Solidifier</style> with a <style color="0xafff00" underlined="true" link="gtceu:misc/mold" hover-info="click link">casting mold (ball)</style>, 
+        consume 1 glass material for every 1 glass tube produced. Glass tubes can be made with only LV voltage.
+        <br/>
+        <br/>
+        Glass tubes can only be used to craft <style color="0xafff00" underlined="true" link="gtceu:circuits/vacuum_tube" hover-info="click link">Vacuum Tube</style>, as a primary material for your first circuit.
```

Comment:
`Glass tubes can be used to craft Vacuum Tubes, a primary component of your first circuit.`